lib/profile/commands/configure.rb

```diff
@@ -31,14 +31,31 @@ def display_details
       def ask_questions
         raise "No valid cluster types available" if !Type.all.any?
         type = cluster_type
+        smart_log = Logger.new(File.join(Config.log_dir,'configure.log'))
         @answers = prompt.collect do
           type.questions.each do |question|
             key(question.id).ask(question.text) do |q|
-              if type.fetch_answer(question.id)
-                q.default type.fetch_answer(question.id)
-              elsif question.default
-                q.default question.default
+
+              prefill = type.fetch_answer(question.id)
+              if question.default_smart
```

Comment:
If the question already has an answer, and we know we'll be using that answer in the `default` field, we don't need to run the `default_smart` command.

Reply:
Addressed in newest version